### ISSUE TITLE
When will the template be upgraded to .Net 7 (or 8)

### ISSUE THREAD

**Issue by ChrisSchaller** [Garderoben/MudBlazor.Templates#389]
Templates is a great way to communicate best practises and project setup, however these templates use the v6 runtimes of MudBlazor (and .Net) and the upgrade to .net 7 leaves a bit of a mess. Can we please have updated versions of the templates for the current runtimes?

**Comment by ScarletKuro** [comment]
Pull requests are welcome in this direction

**Comment by mahald** [comment]
I got a .net 8 Web App Template here: https://github.com/mahald/MudBlazor.Template.WebApp. Auth is working but not in MudBlazor yet everything else is working as it is in the Microsoft template.

**Comment by mahald** [comment]
Added Pull Request  https://github.com/MudBlazor/Templates/pull/410

**Comment by MatejMarecek** [comment]
How can I use the new template? I assume that it was not released yet?

When I do `$ dotnet new install MudBlazor.Templates`, I will get

```text
The following template packages will be installed:
   MudBlazor.Templates

Success: MudBlazor.Templates::0.6.3 installed the following templates:
Template Name        Short Name  Language  Tags
-------------------  ----------  --------  --------------------------------------
MudBlazor Templates  mudblazor   [C#]      Blazor/Desktop/macOS/Windows/Cloud/Web
```

Running `$ dotnet new mudblazorwebapp --interactivity Auto --auth Individual` results in:

```text
No templates or subcommands found matching: 'mudblazorwebapp'.

To list installed templates similar to 'mudblazorwebapp', run:
   dotnet new list mudblazorwebapp
To search for the templates on NuGet.org, run:
   dotnet new search mudblazorwebapp
```

**Comment by ScarletKuro** [comment]
> How can I use the new template? I assume that it was not released yet?

No, it was not yet released (https://github.com/MudBlazor/Templates/pull/410#issuecomment-1834131987)

You can clone the repository and install it this way
`dotnet new install ./Templates/src/MudBlazorWebTemplate`

**Comment by jeffward01** [comment]
> > How can I use the new template? I assume that it was not released yet?
> 
> No, it was not yet released ([#410 (comment)](https://github.com/MudBlazor/Templates/pull/410#issuecomment-1834131987))
> 
> You can clone the repository and install it this way `dotnet new install ./Templates/src/MudBlazorWebTemplate`

Thanks, this worked for me.  However it does not work in Rider. It only works in Visual Studio. This may be due to Rider not fully supporting `net8.0` yet.

The error I got in Rider was `AppName.Client.exe` does not exist.  When I switched to VS it loaded just fine :)

**Comment by Garderoben** [comment]
They are updated and released now with some changes, check the new readme.
